app/packages/core/src/components/Filters/NumericFieldFilter.tsx

```diff
@@ -157,6 +164,12 @@ const NumericFieldFilter = ({
     (nonfinites.length === 1 && nonfinites[0][0] === "none")
   );
 
+  const hasNone =
+    nonfinites.some((x) => x[0] === "none") &&
+    nonfinites.find((x) => x[0] === "none")![1].count !== 0;
```

Comment:
Based on the implementation of `useNonfinites` empty counts are already filtered:
```suggestion
  const hasNone = nonfinites.some((x) => x[0] === "none")
```